README.md

```diff
@@ -94,7 +94,61 @@ playlist_videos = supadata.youtube.playlist.videos(
 )
 print(f"Regular videos: {playlist_videos.video_ids}")
 print(f"Shorts: {playlist_videos.short_ids}")
-```
+
+# --- Batch Operations ---
+
+# Start a batch job to get transcripts for multiple videos
+# You can provide video IDs, a playlist ID, or a channel ID
+transcript_batch_job = supadata.youtube.transcript.batch(
+    video_ids=["dQw4w9WgXcQ", "xvFZjo5PgG0"],
+    # playlist_id="PLlaN88a7y2_plecYoJxvRFTLHVbIVAOoc", # alternatively
+    # channel_id="UC_9-kyTW8ZkZNDHQJ6FgpwQ", # alternatively
+    lang="en",  # Optional: specify preferred transcript language
+    limit=100   # Optional: limit for playlist/channel
+)
+print(f"Started transcript batch job: {transcript_batch_job.job_id}")
+
+# Start a batch job to get video metadata for a playlist
+video_batch_job = supadata.youtube.video.batch(
+    playlist_id="PLlaN88a7y2_plecYoJxvRFTLHVbIVAOoc",
+    limit=50
+)
+print(f"Started video metadata batch job: {video_batch_job.job_id}")
+
+# Get the results of a batch job (poll until status is 'completed' or 'failed')
+import time
+
+job_id = transcript_batch_job.job_id # or video_batch_job.job_id
+
+while True:
+    try:
+        batch_results = supadata.youtube.batch.get_batch_results(job_id=job_id)
+        print(f"Job {job_id} status: {batch_results.status}")
+
+        if batch_results.status == "completed":
+            print(f"Job completed at: {batch_results.completed_at}")
+            print(f"Stats: Total={batch_results.stats.total}, Succeeded={batch_results.stats.succeeded}, Failed={batch_results.stats.failed}")
+            for item in batch_results.results:
+                if item.error_code:
+                    print(f"  Video {item.video_id} failed with error: {item.error_code}")
+                elif item.transcript:
+                    print(f"  Video {item.video_id} transcript ({item.transcript.lang}): {len(item.transcript.content)} chunks/chars")
+                elif item.video:
+                     print(f"  Video {item.video_id} metadata: Title='{item.video.title}'")
+            break
+        elif batch_results.status == "failed":
+            print(f"Job {job_id} failed.")
+            # You might want to add more error details if available in a real scenario
+            break
+        elif batch_results.status in ["queued", "active"]:
+            print("Waiting for job to complete...")
+            time.sleep(5) # Poll every 5 seconds
+        else:
+            print(f"Unknown job status: {batch_results.status}")
+            break
+    except SupadataError as e:
+        print(f"Error fetching batch results for job {job_id}: {e}")
+        break
```

Comment:
this example is much too long. 

let's keep it to max 5 or so lines, just like other snippets.